app/lib/services.rb

```diff
@@ -20,6 +20,7 @@ def self.content_store
 
   def self.link_checker_api
     @link_checker_api ||= GdsApi::LinkCheckerApi.new(
-      Plek.new.find("link-checker-api"))
+      Plek.new.find("link-checker-api")
+      )
```

Comment:
As you're reformatting this chunk, the closing `)` needs to be out-dented by two characters.